client/components/section-nav/test/index.jsx

```diff
@@ -1,11 +1,21 @@
-var assert = require( 'chai' ).assert,
-	sinon = require( 'sinon' ),
-	useMockery = require( 'test/helpers/use-mockery' ),
-	useFakeDom = require( 'test/helpers/use-fake-dom' ),
-	ReactDom, React, TestUtils, SectionNav;
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import { noop } from 'lodash';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+let	ReactDom, React, TestUtils, SectionNav;
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/8100/commits/c50e489fac6b4c357c90d6bc016fd531cea538ca and put in another open PR :)